index.css

```diff
@@ -322,3 +322,32 @@ body {
 .feed-section > .content > .payment > img {
   margin-top: 60px;
 }
+/*================================================================================================*/
+/*=================================== 4 - TESTIMONIALS-SECTION ===================================*/
+/*================================================================================================*/
+.testimonials-section {
+  padding: 140px 180px 0px 180px;
+  position: relative;
+}
+.testimonials-section > h3 {
+  margin: 0px;
+  text-align: center;
+  margin-bottom: 47px;
+}
+.testimonials-section .cards > * {
+  display: inline-block;
+  max-width: calc(100% / 4 - 3px);
+  margin-bottom: 21px;
+}
+.testimonials-section .indicator {
+  margin: 0 auto;
+  margin-bottom: 32px;
+}
+.testimonials-section button {
+  margin-bottom: 180px;
```

Comment:
do we need to have ```margin-bottom``` here? we already have padding-bottom in the ```.testimonials-section {```

Reply:
In .testimonials-section { I have all another paddings, except bottom, but yes, a can remove margin from button, and add padding to all section.